gpt4all-chat/tests/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ add_test(NAME ChatPythonTests
     COMMAND ${Python3_EXECUTABLE} -m pytest --color=yes "${CMAKE_CURRENT_SOURCE_DIR}/python"
 )
```

Comment:
Now that we are running more python tests, I'm realizing that running pytest *as* a CTest is not ideal, because its progress output is captured by CTest and only displayed on failure (at least, for `make check`). I think the check target should depend on ctest and pytest separately.